docs/whats-new/preview-4.md

```diff
@@ -604,15 +604,21 @@ Tool authors can support this very small set of resource types to model lots of
 
+## Azure Developer CLI (azd)
+
+The [Azure Developer CLI](https://aka.ms/azd) (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing. All Azure resources can be described using a bicep module.
+
+In addition to new features, azd will automatically create secrets in Azure Container Apps for any parameter marked as a secret, and any environment variable that references a secret or uses a connection string.
+
 ### Visual Studio Publish to Azure
 
-This release we've enabled a Visual Studio publish experience for .NET Aspire applications.
+This release we've enabled a Visual Studio publish experience for .NET Aspire applications. This new deployment experience enables a single right-click-publish gesture that results in *all* of the nodes of your .NET Aspire project being published as individual Azure Container Apps in an Azure Container Apps Environment. We've chosen Azure Container Apps as the premier container PaaS destination for .NET Aspire apps as it enables orchestrated containerized deployments of multi-node microservice and distributed monolith applications. Even better - the publishing experience in Visual Studio uses `azd` under the hood, so you'll be able to publish .NET Aspire apps directly from Visual Studio to any `azd` environment you have running in Azure. 
 
-## Azure Developer CLI
 
-The [Azure Developer CLI](https://aka.ms/azd) (azd) has been updated to support the new manifest types introduced in preview 4. Azd supports prompting for parameter resources, and supports the bicep resource which enables deploying any thing. All Azure resources can be described using a bicep module.
+![.NET Aspire right-click publish in Visual Studio](media/preview-4/aspire-azd-publish.gif)
 
-In addition to new features, azd will automatically create secrets in Azure Container Apps for any parameter marked as a secret, and any environment variable that references a secret or uses a connection string.
+
+With so many similarities between Container Apps' support for [add-on services](https://learn.microsoft.com/azure/container-apps/services) like PostgreSQL, Kafka, and Redis and the support offered by mirroring .NET Aspire components ([PostgreSQL](https://learn.microsoft.com/dotnet/aspire/database/postgresql-component?tabs=dotnet-cli), [Kafka](https://learn.microsoft.com/dotnet/aspire/messaging/kafka-component?tabs=dotnet-cli), and [Redis](https://learn.microsoft.com/dotnet/aspire/caching/stackexchange-redis-component?tabs=dotnet-cli), for example), targeting Azure Container Apps as our **first** publishing destination was an easy decision. You'll be able to use Azure Container Apps as your development publishing target out-of-the-box using Visual Studio Preview's upcoming 17.10 preview 3 release. 
 
 ### Prompting for parameters
```

Comment:
@bradygaster I think given the VS experience doesn't support inputs for Parameter in this preview, maybe a remark in this section?

Reply:
good point - i need to mention the AZD auth login part too. thanks!

Reply:
lgtm